### DISCUSSION TITLE
Welcome to chinese-independent-blogs Discussions!

### DISCUSSION THREAD

**Discussion by timqian** [timqian/chinese-independent-blogs#1109] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timqian** [comment]
test: 欢迎来到独立博客的世界

**Comment by Kanna-Neko** [comment]
## share questions
1. 当一个人想要分享自己文章或者自己的博客的时候, 如何确保一定的质量.

## share ideas
1. 上面的问题, 提供一个解决方法, 可以考虑白名单.
2. 考虑到是发电项目, 个人认为最好的方法就是写一个RSS推荐工具, 每次运行的时候从白名单中个性化推荐一些博客.

**Reply by timqian** [reply]
我也在思考这个问题，做一个类似 https://bloghub.fun/ 这样的网站，并且加上类似 hackernews 的点赞排序功能，可能可以解决这个问题
